src/server.cpp

```diff
@@ -594,7 +594,8 @@ void Server::AsyncRunStep(bool initial_step)
 		// Run Map's timers and unload unused data
 		ScopeProfiler sp(g_profiler, "Server: map timer and unload");
```

Comment:
maybe you should correct the code style of the lines above and below this, whilst you're here?